pkgs/development/python-modules/django-autocomplete-light/default.nix

```diff
@@ -0,0 +1,76 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, django
+, six
+
+, pytestCheckHook
+, coverage
+, django-debug-toolbar
+, django-extensions
+, django-taggit
+, django_tagging
+, mock
+, pytest-django
+, selenium
+, splinter
+, sqlparse
+, tenacity
+, whitenoise
+}:
+
+buildPythonPackage rec {
+  pname = "django-autocomplete-light";
+  version = "3.9.4";
+
+  src = fetchFromGitHub {
+    owner = "yourlabs";
+    repo = "django-autocomplete-light";
+    rev = version;
+    sha256 = "sha256-YUiGN6q7ARM/rg7d+ykeDEYZDYjB+DHxMCmdme6QccU=";
+  };
+
+  propagatedBuildInputs = [
+    django
+    six
+  ];
+
+  # Too many un-packaged dependencies
+  doCheck = false;
```

Comment:
Did not investigate this question. Will come back to you once I have checked.

Reply:
Nope, I get a `ModuleNotFoundError` because `django-querysetsequence` (a transitive dependency) is not installed, and not packaged by us. 

See https://github.com/yourlabs/django-autocomplete-light/issues/971 for a similar issue.